extension-readme.md

```diff
@@ -1,134 +1,97 @@
 
-![Configure and Run 1](./resources/marketplace/sam-configure-and-run-still-1.png)
+![Add Debug Configuration Direct](./resources/marketplace/Codelens-direct-function.png)
 
-Take special note of the _CodeLenses_ that enable you to run and debug the application locally.
+###### The _CodeLens_ indicator in the application file lets you add a debug configuration for a selected AWS Lambda function.
 
-The status and results are shown in the **OUTPUT** panel when the **AWS Toolkit** output channel is selected.
+When you run a debug session, the status and results are shown in the **OUTPUT** panel when the **AWS Toolkit** output channel is selected.
 
 ![Configure and Run 1](./resources/marketplace/sam-configure-and-run-still-2.png)
 
-When you're satisfied, you can [deploy your application](https://docs.aws.amazon.com/console/toolkit-for-vscode/deploy-serverless-app) to a CloudFormation stack and then [run the Lambda function](https://docs.aws.amazon.com/console/toolkit-for-vscode/remote-lambda) on AWS.
+###### After a local run is complete, the output appears in the **OUTPUT** tab.
+
+When you're satisfied with performance, you can [deploy your serverless application](https://docs.aws.amazon.com/console/toolkit-for-vscode/deploy-serverless-app). The SAM template is converted to a CloudFormation template, which is then used to deploy all the application's assets to the AWS Cloud.
 
-For full details, see the [AWS Explorer](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/aws-explorer.html) in the Toolkit's user guide.
+### Supported runtimes
 
-#### Additional Information about CodeLenses
+The Toolkit _local SAM debugging_ feature supports these runtimes:
 
-The functions that have Codelenses are those that use AWS Lambda-function handler syntax. A _handler_ is a function that Lambda calls to start execution of a Lambda function. These CodeLenses enable you to locally run or debug the corresponding serverless application. CodeLens actions in the Toolkit include:
+-   C# (.NET Core 2.1)
+-   JavaScript (Node.js 10.x, 12.x)
+-   Python (2.7, 3.6, 3.7, 3.8)
 
--   **Configure**, for specifying function configurations such as an event payload and environment variable overrides.
--   **Run Locally**, for running the function _without_ debugging.
--   **Debug Locally**, for running the function _with_ debugging.
+For more information see [Working with AWS Serverless Applications](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/serverless-apps.html) in the user guide.
 
-{ [Return to Features](#features) } or { [Return to Top](#top) }
+{ [Return to Top](#top) }
 
 ---
 
-## <a id="cdk-apps"></a>AWS CDK Applications
+## <a id="cdk-apps"></a>AWS CDK applications
```

Comment:
Section title is no longer `Title Case`?

Reply:
restored title case for everything except "task definition files".